game/menu.cc

```diff
@@ -92,10 +90,10 @@ void Menu::action(int dir) {
 }
 
 void Menu::clear(bool save_root) {
-	if (!save_root) root_options.clear();
+	if (!save_root) root_options->clear();
 	menu_stack.clear();
 	selection_stack.clear();
-	menu_stack.push_back(&root_options);
```

Comment:
not sure this is realy what we want to have: should menu_stack **own** the root_options? It seems to me the sub menu is just using it, so using a raw pointer can be ok (or reference wrapper if you really don't like pointers), or if you really fear dangling pointers, you may use weak_ptr here.